modules/backend/lang/en/lang.php

```diff
@@ -320,6 +320,7 @@
         'missing_model' => 'Relation behavior used in :class does not have a model defined.',
         'invalid_action_single' => 'This action cannot be performed on a singular relationship.',
         'invalid_action_multi' => 'This action cannot be performed on a multiple relationship.',
+        'relationwidget_unsupported_type' => 'This relation type is unsupported by the Relation Widget: :type',
```

Comment:
@RomainMazB this phrasing reads a little better - would you mind using this and adjust the French translation to suit? :)

```suggestion
        'relationwidget_unsupported_type' => 'The ":type" relation type is unsupported by the Relation widget',
```